packages/cli/src/daemon-config.ts

```diff
@@ -194,6 +194,13 @@ export class DaemonAnchorConfig {
   @jsonMember(String, { name: 'anchor-service-url' })
   anchorServiceUrl?: string
 
+  /**
+   * Controls the authentication method Ceramic uses to make requests to the Ceramic Anchor Service.
+   * When specifying in a config file, use the name 'auth-method'.
+   */
+  @jsonMember(String, { name: 'auth-method' })
```

Comment:
right now it is just `did` or empty string for unauthenticated. a node operator might change this depending on what anchor service they are using.

Reply:
we only support one anchor service per network though - can we make the decision automatically and reduce how much configuration we expose to node operators?

Reply:
I think this is a good way to easily disable authenticating every request. I'm not really positive when this would be useful, in some potential debugging or failure scenario, but it feels like a good idea to have an easy way to turn this off. 

Don't feel too strongly though either way.

Reply:
well actually this could be a good way to feature flag also.

Reply:
this is proving useful because we want auth disabled for Clay. keeping unless any other objections come up